Analysis/Tasks/PWGHF/taskDPlus.cxx

```diff
@@ -26,23 +26,28 @@ using namespace o2::aod::hf_cand_prong3;
 
 /// D± analysis task
 struct TaskDPlus {
-  OutputObj<TH1F> hmass{TH1F("hmass", "3-prong candidates;inv. mass (#pi K #pi) (GeV/#it{c}^{2});entries", 500, 1.6, 2.1)};
+  OutputObj<TH1F> hmass{TH1F("hmass", "3-prong candidates;inv. mass (#pi K #pi) (GeV/#it{c}^{2});entries", 350, 1.7, 2.05)};
   OutputObj<TH1F> hptcand{TH1F("hptcand", "3-prong candidates;candidate #it{p}_{T} (GeV/#it{c});entries", 100, 0., 10.)};
+  OutputObj<TH1F> hEta{TH1F("hEta", "3-prong candidates;candidate #it{#eta};entries", 100, -2., 2.)};
+  OutputObj<TH1F> hCt{TH1F("hCt", "3-prong candidates;proper lifetime (D^{#pm}) * #it{c} (cm);entries", 120, -20., 100.)};
+  OutputObj<TH1F> hdeclength{TH1F("hdeclength", "3-prong candidates;decay length (cm);entries", 200, 0., 2.)};
+  OutputObj<TH1F> hdeclengthxy{TH1F("hdeclengthxy", "3-prong candidates;decay length xy (cm);entries", 200, 0., 2.)};
+  OutputObj<TH1F> hnormdlxy{TH1F("hnormdlxy", "3-prong candidates;norm. decay length xy;entries", 80, 0., 80.)};
+  OutputObj<TH1F> hCPA{TH1F("hCPA", "3-prong candidates;cos. pointing angle;entries", 110, -1.1, 1.1)};
+  OutputObj<TH1F> hCPAxy{TH1F("hCPAxy", "3-prong candidates;cos. pointing angle xy;entries", 110, -1.1, 1.1)};
+  OutputObj<TH1F> himpparxy{TH1F("himpparxy", "3-prong candidates;impact par. xy (cm);entries", 200, -1., 1.)};
+  OutputObj<TH1F> hnormip{TH1F("hnormip", "3-prong candidates;norm. IP;entries", 200, -20., 20.)};
+  OutputObj<TH1F> himpparsumsq{TH1F("himpparsumsq", "3-prong candidates;sq. sum of prong imp. par. (cm^2);entries", 100, 0., 1.)};
+  OutputObj<TH1F> hDecLenErr{TH1F("hDecLenErr", "3-prong candidates;decay length error (cm);entries", 100, 0., 1.)};
+  OutputObj<TH1F> hDecLenXYErr{TH1F("hDecLenXYErr", "3-prong candidates;decay length xy error (cm);entries", 100, 0., 1.)};
+  OutputObj<TH1F> hImpParErr{TH1F("hImpParErr", "3-prong candidates;impact parameter error (cm);entries", 100, 0., 1.)};
```

Comment:
Why are the histograms reshuffled?
Please fix the names of the new histograms to comply with the conventions.
I will take care of the old ones.

Reply:
This order just makes more sense to me, no specific reason.
For the histogram names, I followed the style already in place. What is the convention?

Reply:
https://github.com/AliceO2Group/CodingGuidelines

Reply:
I had already checked the guidelines. Do you mean the [generic naming rules](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=General_Naming_Rules#General_Naming_Rules)? So, for example, `himpparxy` -> `hImpParXY` (I would keep the abbreviations to avoid very long variables)

Reply:
Yes